pkg/diffs/diffs_test.go

```diff
@@ -81,7 +81,7 @@ func TestGetChangedCiopConfigs(t *testing.T) {
 	for _, tc := range testCases {
 		t.Run(tc.name, func(t *testing.T) {
 			before, after := tc.configGenerator()
-			actual := GetChangedCiopConfigs(before, after, logrus.NewEntry(logrus.New()))
+			actual, _ := GetChangedCiopConfigs(before, after, logrus.NewEntry(logrus.New()))
```

Comment:
If all the changes in `_test.go` files are to add `_` and `nil` I think we both know what's missing ;)

Reply:
I [WIP] because of that. I will take care of it.